doc/classes/ProjectSettings.xml

```diff
@@ -830,6 +830,16 @@
 			[b]Note:[/b] V-Sync modes other than [b]Enabled[/b] are only supported in the Forward+ and Mobile rendering methods, not Compatibility.
 			[b]Note:[/b] This property is only read when the project starts. To change the V-Sync mode at runtime, call [method DisplayServer.window_set_vsync_mode] instead.
 		</member>
+		<member name="display/window/vsync/buffer_count" type="int" setter="" getter="" default="2">
+			Sets the number of buffers before stalling to wait for the GPU.
+			2 corresponds to double buffering and 3 to triple buffering.
+			Single buffering may give you the lowest lag but at the high cost of no parallelism between CPU &amp; GPU
+			Assuming you're able to reach 60fps all the time, then double buffering gives you the lowest lag.
+			Use Triple buffering if V-Sync is enabled and the GPU is struggling to keep 60 fps (or whatever the monitor refresh rate is).
+			Triple buffering is almost a must if you're using V-Sync MAILBOX mode.
+			[b]Note:[/b] This setting is not supported by all APIs.
+			[b]Note:[/b] This property is only read when the project starts.
```

Comment:
Every line break creates a new paragraph, so I suggest using those sparingly.

```suggestion
			Sets the number of buffers before stalling to wait for the GPU. A higher values allows for higher framerates in some situations at the cost of higher visual latency. 2 corresponds to double buffering and 3 to triple buffering. Single buffering may give you the lowest latency but at the high cost of no parallelism between CPU &amp; GPU
			Assuming you're able to reach the monitor's refresh rate all the time, double buffering gives you the lowest latency while still having some level of parallelism.
			Use Triple buffering if V-Sync is enabled and the GPU is struggling to keep up with the monitor's refresh rate.
			Triple buffering is almost a must if you're using the Mailbox V-Sync mode.
			[b]Note:[/b] This setting is only supported with Vulkan-based rendering methods (Forward+ and Mobile), not Compatibility.
			[b]Note:[/b] This property is only read when the project starts. There is currently no way to change this value at run-time.
```

Also, are the terms "double buffering" and "triple buffering" correct here? I see those being used to refer to V-Sync behavior, with double buffering offering less latency but a "permanent" drop to 30 FPS (or 20 FPS, 15 FPS, …) whenever you can't sustain 60 FPS. I thought this was configured separately, see https://github.com/godotengine/godot-proposals/issues/4065.